src/learning-catalogue/model/fileModule.ts

```diff
@@ -4,17 +4,29 @@ import {IsNotEmpty, IsPositive} from 'class-validator'
 export class FileModule extends Module {
 	@IsNotEmpty({
 		groups: ['all', 'url'],
-		message: 'validation.module.url.empty',
+		message: 'validation_module_url_empty',
```

Comment:
Are we going for `_` or `.` in such case? It seems to be a mixture of both at the moment. I think my personal preference would be `.` but happy to be convinced otherwise.